server.js

```diff
@@ -1,33 +1,77 @@
-import express from 'express'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
-
-// Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(express.json())
-
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
-})
-
-// Start the server
+import express from "express";
+import cors from "cors";
+import data from "./data.json";
+
+// Port the app will run on. Defaults to 8080.
+const port = process.env.PORT || 8080;
+const app = express();
+
+// Enables cors and json body parsing
+app.use(cors());
+app.use(express.json());
+
+// Start route on default port
+//sending a html-file as the response by using sendFile
+app.get("/", (req, res) => {
+  res.sendFile(__dirname + "/index.html");
+});
+
+//the style-sheet is linked in its own route, and sent as the response using sendFile
+app.get("/style.css", (req, res) => {
+  res.sendFile(__dirname + "/style.css");
+});
+
+// First route
+app.get("/seasons", (req, res) => {
+  res.json(data);
+});
+
+// Second route
+app.get("/seasons/:season", (req, res) => {
+  const seasonId = Number(req.params.season);
+
+  const season = data.find((s) => s.season === seasonId);
+
+  if (!season) {
+    res.status(404).send("no season found with that id");
+  } else {
+    res.json(season);
+  }
+});
+
+// Third route
+app.get("/seasons/:season", (req, res) => {
+  const seasonWinner = req.params.season;
+
+  const season = data.find((s) => s.season === Number(seasonId));
+
+  // 404 error if the id does not match
+  if (!season) {
+    res.status(404).send("no season found with that id");
+  } else {
+    res.json(season);
+  }
+});
+
+//this is a dummy
+
+app.get("/compute/complex", (req, res) => {
```

Comment:
Not sure I understood this part? I understand it's a dummy but would be interesting to know a bit more. :)